### PR TITLE
Update to ActronAir Neo plugin

### PR DESCRIPTION
**Pull request by domalab**
Hi @bstillitano, @jxg81 

I have made some updates to ActronAir Neo plugin. 

@jxg81 I noticed that you made updates to your Que plugin. I'm looking if it can be integrated into the Neo plugin.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (bstillitano) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Why are we removing the issue templates? Please leave these intact

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (bstillitano) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: Why are we removing the issue templates? Please leave these intact

File: .github/workflows/push.yml
Comment (bstillitano) [inline .github/workflows/push.yml:1]: Why are we removing the issue templates? Please leave these intact

**Review by bstillitano — APPROVED** [review]

**Comment by bstillitano** [comment]
@domalab this is merged, a massive thank you for your efforst! A few notes below for future contributions and general housekeeping:

1. I've undone the renaming from `homebridge-actron-neo` to `homebridge-actronair-neo`. This needs to remain as is for NPM packages and homebridge versioning.
2. I've undone the change you made which pointed the readme/npm files to your repo. Whilst I 110% appreciate your efforts, changing links in this repo to point to an external repo are a huge security risk.
3. I've undone the change you made which changed the plugin author to yourself. Your contribution and efforts are recognised in the public contributors section of the repo.
4. I've re-added all of the git issue templates, these are important for ensuring we get meaningful and reproducible bug reports.
5. I've restored the badges that were removed from the `README` file.
6. I've re-added the workflows that were deleted, these automate our NPM releases and git tags.
7. I've fixed the eslint errors that were preventing an NPM release from being made.
8. I've re-added the `package-lock.json` file, this is a hard requirement for NPM releases.
9. I've re-added the `.eslintrc.json` file, this is a hard requirement for NPM releases.

**Comment by domalab** [comment]
Thanks for the comments / feedback. 

Didn't know that some of the changes break things. 

Good to know as I'm still learning 😊

**Pull request merged**